pmd-apex/src/main/resources/rulesets/apex/ruleset.xml

```diff
@@ -1,444 +1,186 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<ruleset xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Default ruleset used by the CodeClimate Engine for Salesforce.com Apex" xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
-   <description>Default ruleset used by the Code Climate Engine for Salesforce.com Apex</description>
+<ruleset xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Default ruleset for Salesforce.com Apex" xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
+   <description>Default ruleset for Salesforce.com Apex</description>
```

Comment:
This ruleset should probably be renamed `quickstart.xml` for consistency with the Java one. Not in this PR though. I'll open an issue to track that.

Reply:
No need for an extra issue. I'll do that in my corrections.

Reply:
Regarding your above comments:

- This PR brings PMD where it was before CodeClimate wise. I never heard that anyone in the Salesforce world uses it so its fair to clean up the mess I made. I have no interest in improving anything Code Climate related. Let that do others...
- I will add the additional consequences to the release notes doc.